src/HttpHandlers/LfdsUsernamePasswordHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Laserfiche.Api.Client.HttpHandlers
+{
+    public class LfdsUsernamePasswordHandler : IHttpRequestHandler
+    {
+        private string _accessToken;
+        private readonly string _username;
+        private readonly string _password;
+        private readonly string _baseUri;
+        private readonly string _repoID;
+        private readonly string _organization;
+        private readonly HttpClient _credentialsClient;
+
+        public LfdsUsernamePasswordHandler(string username, string password, string organization, string repoID, string baseUri, HttpClient client = null, string token = null)

```

Comment:
Is the HttpClient and token parameters necessary? Should try to keep this consistent with the other HttpRequestHandler and only add the mandatory arguments

Also in this library, the parameter is called `baseUri`, but in the repo client library, the parameter is `baseUrl`. Should be consistent. I guess we use `baseUrl` in other places, so let's change it here